tools/dep-device-details.sh

```diff
@@ -5,7 +5,7 @@
 DEP_ENDPOINT=/devices
 URL="${BASE_URL}/proxy/${DEP_NAME}${DEP_ENDPOINT}"
 
-jq -n --arg device "$1" '.devices = [$device]' \
+jq -n --arg devices "$*" '{devices: ($devices|split(" "))}' \
```

Comment:
Nice! TIL `jq` has a `split`! Too bad it doesn't trim surrounding whitespace, too.